--- firmwarelist.sh	2022-09-28 23:07:51.174202713 +0200
+++ firmwarelist.sh.new	2022-09-28 23:07:53.104188578 +0200
@@ -48,7 +48,7 @@
 			fi
 
 			local PACKAGES
-			PACKAGES="$(gunzip -c "${CONTENTS_FILE}" | awk '/^lib\/firmware/ { print $2 }' | sort -u )"
+			PACKAGES="$(gunzip -c "${CONTENTS_FILE}" | awk '/^lib\/firmware/ { print $2 }' | grep -v 'nvidia' | sort -u )"
 			FIRMWARE_PACKAGES="${FIRMWARE_PACKAGES} ${PACKAGES}"
 
 			# Don't waste disk space, if not making use of caching
